doc/releases.md

```diff
@@ -0,0 +1,51 @@
+# Releases
+
+https://github.com/badges/shields is a monorepo and hosts the Shields frontend and server code as well as the [badge-maker][npm package] NPM library (and the [badge design specification](https://github.com/badges/shields/tree/master/spec)). The packaging and release processes for these respective items is described below.
+
+## badge-maker package
+
+We follow [Semantic Versioning](https://semver.org/) for the badge-maker package, and publish git Tags using the form `X.Y.Z`, and all such tags of this form are badge-maker releases (e.g. [3.3.0](https://github.com/badges/shields/releases/tag/3.3.0)
+
+The [badge-maker][npm package] is also published to the npm.js registry.
+
+Releases of badge-maker are done as and when needed, and not on any predetermined interval.
+
+## Shields.io service
+
+The [Shields.io Service][shields.io] consists of the frontend [https://shields.io][shields.io] website which allows users to browse and discover available badges, as well as the badge server backend that serves up requested badges (e.g. https://img.shields.io/badge/badges-rock-blue).
+
+This is the core, free, anonymous service available for anyone to use and which serves up more than 500 million badges every month!
+
+We do not have a fixed schedule for deploying updates to the Shields.io production environment, but we typically deploy the latest version at least once per week.
+
+More information about the production environment can be found [here][production hosting]
+
+## Shields server
+
+Some users may wish to host their own instance of Shields so we also make it possible for users to do so. This is particularly useful if you want to serve badges for resources that require authentication or are not exposed to the internet (e.g: inside a corporate network). A variety of options are available either by installing from source or using a docker image.
+
+### Important information
+
+This Shields server is the exact same codebase that powers the main Shields.io service; we do not have a separate self-hosted version of Shields. This means that the server codebase is geared towards the development, maintenance, and operation of the Shields.io service so there are a few things to note:
+
+- We often have to reject or de-prioritize feature requests that are only applicable for self-hosting and/or which may be problematic for the core Shields.io offering (e.g. requiring auth on the badge server)
```

Comment:
I think in the ops meeting we said something about we're happy to host docs/tutorials around some of this stuff. We just won't usually merge PRs to core for features that are only applicable to self-hosted instances or irrelevant for users of shields.io itself. Shall we add that here.

Reply:
Yeah I think this is the right general area to call that out. I'll have a go and see if that works better inline literally here or if it'd be worthwhile to have a block of its own, as I'm thinking it'd be good to actually solicit that type of feedback/story docs

Reply:
Added as a separate paragraph in the same section. Let me know what you think, e.g. if it'd be better as another bullet item, etc.